bin/infrasim-main

```diff
@@ -60,6 +60,13 @@ if __name__ == '__main__':
             print "Restart InfraSIM service..."
             socat.start_socat()
             ipmi.start_ipmi()
+        elif sys.argv[1] == "version":
+            print "Kernel:  ", run_command("uname -sr")[1].split('\n')[0]
+            print "Base OS: ", run_command("cat /etc/issue")[1].split('\\')[0]
+            print "QEMU:    ", run_command("qemu-system-x86_64 --version")[1].split(',')[0]
+            print "OpenIPMI:", run_command_anyway("ipmi_sim -v")[1].split('\n')[0]
+            print "Socat:   ", ' '.join(run_command("socat -V")[1].split('\n')[1].split(' ')[0:3])
```

Comment:
As talked, since binary in our system may vary due to user. We shall display binary version of what InfraSIM actually uses. We define get_qemu(), get_ipmi(), get_socat(), can be your reference.